### PR TITLE
fix(odd): address design QA feedback Touchscreen Brightness

### PR DESCRIPTION
**Pull request by koji**
# Overview
address design qa feedback
RCORE-564

The blue color is using the old name but this will be updated once https://github.com/Opentrons/opentrons/pull/12430 is merged into the `edge`

[feedback from Mel]
- 24px margin between buttons and the purple rectangles 
- Rectangles in the figma design is a bit bigger, the whole set is ~696 and in the code it’s ~600px
- change column gap to 8px, sorry that was my b! 
- Do the +/- buttons have a focus-visible or pressed state? 
- The header is constructed a bit differently than I have in the figma—is this final?  <- This will be fixed by a following PR

[design]
https://www.figma.com/file/AoTLAYuWawlaWItB1umOjr/Opentrons-Flex-Touchscreen-Designs?node-id=7873-118259&t=4ahy6SgWk7GfENQy-0

# Test Plan
Settings -> Touch Brightness
- the colors are the same as design
- +/- button has pseudo class

# Changelog
- Update TouchscreenBrightness to address feedback

# Review requests

# Risk assessment
low

### CODE REVIEW

File: app/src/organisms/OnDeviceDisplay/RobotSettingsDashboard/TouchscreenBrightness.tsx [hunk 1]
```diff
@@ -26,15 +29,26 @@ import {
...
+  }
+
+  &:active {
+    color: ${COLORS.darkBlack_forty};
+  }
+`
+
 interface RectProps {
   isActive: boolean
 }
```
Comment (jerader) [inline app/src/organisms/OnDeviceDisplay/RobotSettingsDashboard/TouchscreenBrightness.tsx]: not introduced in this PR but can we rename this to `BrightnessTileProps` or something more descriptive?

**Review by jerader — APPROVED** [review]
lgtm! just let a comment about naming

File: app/src/organisms/OnDeviceDisplay/RobotSettingsDashboard/TouchscreenBrightness.tsx (hunk 1 above)
Comment (koji) [reply app/src/organisms/OnDeviceDisplay/RobotSettingsDashboard/TouchscreenBrightness.tsx]: Thank you for the suggestion. That sounds better.
I will update the name.

**Review by ewagoner — APPROVED** [review]
lgtm!

**Pull request merged**